custom_dc/README.md

```diff
@@ -32,6 +32,58 @@ Additionally, a custom DC combines its own local datasets with base DC datasets
   You can use a sample key `AIzaSyA9RsPS8tBCKwrET3qqAzydhOMWP0Ee8Y8` for local
   development as discussed below.
 
+## Quick Start
+
+Once the prerequisites and api keys from above are in place, 
+here's how you can start a local custom DC instance quickly.
+
+### Env variables
+
+Open [sqlite_env.list](sqlite_env.list) and specify values for `DC_API_KEY` and `MAPS_API_KEY`.
+
+> NOTE: Do not use double-quotes or spaces when specifying the values.
+
+### Run docker
+
+In the root of this repository, run:
+
+```bash
+docker run -it \
+-p 8080:8080 \
+-e DEBUG=true \
+--env-file $PWD/custom_dc/sqlite_env.list \
+-v $PWD/custom_dc/sample:/userdata \
+-v $PWD/server/templates/custom_dc/custom:/workspace/server/templates/custom_dc/custom \
+gcr.io/datcom-ci/datacommons-website-compose:2023_11_30_1
```

Comment:
can we tag this as v1? Then later on we can just update v1 in the container registry than updating the doc here.

Reply:
How about tagging it as `stable` and mentioning that it points to the latest stable one?

If we tag with a number, it usually indicates something that never changes.

Reply:
sg

Reply:
Done.